rex-release/out.sh

```diff
@@ -47,6 +47,22 @@ upload-release() {
 
   # books files with explicit content type set because they don't have extensions, loaded unversioned so no cloudfront caching
   aws s3 sync --exclude 'service-worker.js' --content-type 'text/html' --cache-control 'max-age=0' "$path/books/" "s3://$bucket/rex/releases/$version/books"
+
+  # configure redirects
+  for row in $(cat $path/rex/redirects.json | jq -r -c '.[]'); do
+    getField() {
+      echo ${row} | jq -r ${1}
+    }
+
+    from=$(echo $(getField '.from'))
+    to=$(echo $(getField '.to'))
+
+    if [ ! -e "$path/$from" ] && [ -e "$path/$to" ]; then
+      aws s3api put-object --bucket "$bucket" --key "$from" --website-redirect-location "$to"
+    else
+      echo "cannot create redirection from $from to $to"
```

Comment:
maybe do 2 loops, one up after the validation on line 40 that will exit with an error, and then skip the conditional down here